packages/react-celo/src/styles.css

```diff
@@ -568,27 +571,78 @@ Number Input spinner overrides
 ===============================
 */
 /* Chrome, Safari, Edge, Opera */
-.use-ck input::-webkit-outer-spin-button,
-.use-ck input::-webkit-inner-spin-button {
+.react-celo input::-webkit-outer-spin-button,
+.react-celo input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
 }
 
 /* Firefox */
-.use-ck input[type='number'] {
+.react-celo input[type='number'] {
   -moz-appearance: textfield;
 }
 
 /*
-Connect Modal
-===========
+Spinner
 */
+.react-celo .react-celo-spinner {
+  animation: react-celo-spinner-rotate 2s linear infinite;
+  width: 50px;
+  height: 50px;
+}
+.react-celo .react-celo-spinner .react-celo-spinner-path {
+  stroke-linecap: round;
+  animation: react-celo-spinner-dash 1.5s ease-in-out infinite;
+}
+
+@keyframes react-celo-spinner-rotate {
+  100% {
+    transform: rotate(360deg);
+  }
+}
 
-.use-ck-connect-container {
-  max-width: calc(min(90vw, 28rem));
-  max-height: calc(min(80vh, 35rem));
-  overflow: auto;
+@keyframes react-celo-spinner-dash {
+  0% {
+    stroke-dasharray: 1, 150;
+    stroke-dashoffset: 0;
+  }
+  50% {
+    stroke-dasharray: 90, 150;
+    stroke-dashoffset: -35;
+  }
+  100% {
+    stroke-dasharray: 90, 150;
+    stroke-dashoffset: -124;
+  }
 }
 
+@media (min-width: 900px) {
+  .react-celo-connect-container {
+    width: 800px;
+    height: 550px;
+  }
+}
+
+@tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  min-height: 100vh;
+  /* mobile viewport bug fix */
+  min-height: -webkit-fill-available;
+}
+
+html {
+  height: -webkit-fill-available;
```

Comment:
This is causing background applied to body to not extend its full height (only to height of window) 

#268